### DISCUSSION TITLE
'GPG error' while following the steps of Prebuilt-MPR's 'Getting started': The repository 'https://proget.makedeb.org prebuilt-mpr Release' is not signed

### DISCUSSION THREAD

**Discussion by DavidGreeg** [makedeb/prebuilt-mpr#356] [Issues]
![Screenshot from 2023-08-26 01-52-31](https://github.com/makedeb/prebuilt-mpr/assets/61276600/12e5fa59-99db-4509-916e-e75d35e7c1dd)
Same error when running sudo update
I'm currently using Debian 12

**Comment by hwittenborn** [comment]
Could you try rerunning the setup instructions @DavidGreeg? It looks like there was some issue when downloading the Prebuilt-MPR's GPG file, though it doesn't appear to be showing what.
